### PR TITLE
Created class Machine and method toWork() and stopWork(),

### PR DESCRIPTION
**Pull request by Eugene-Yukhimenko**
also creted classes Truck, Bulldozer, Excavator that will inherit from Machine. In those classes overrided doWork() and stopWork(), also in MainApp class created Machine array with Truck, Excavator, Bulldozer and called methods doWork() and stopWork() in a loop.

### CODE REVIEW

File: src/main/java/MainApp.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        Machine[] machine = new Machine[] {excavator, bulldozer, truck};
+        for (Machine machinery : machine) {
+            machinery.doWork();
+            machinery.stopWork();
+        }
+
+    }
+}
+
+
```
Comment (olekskov) [inline src/main/java/MainApp.java:16]: 2 unnecessary empty lines

File: src/main/java/MainApp.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+public class MainApp {
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
+        Truck truck = new Truck();
+
+        Machine[] machine = new Machine[] {excavator, bulldozer, truck};
```
Comment (olekskov) [inline src/main/java/MainApp.java:7]: Variables for single usage are rarely required
```suggestion
        Machine[] machine = new Machine[] {new Excavator(), new Bulldozer(), new Truck()};
```

**Review by olekskov — APPROVED** [review]
Good job, approved.
Please check comments below for further tasks

**Comment by Eugene-Yukhimenko** [comment]
open now

**Comment by Eugene-Yukhimenko** [comment]
open now

**Comment by Eugene-Yukhimenko** [comment]
open now

**Pull request closed without merging**